x-pack/plugins/synthetics/kibana.json

```diff
@@ -1,6 +1,6 @@
 {
   "configPath": ["xpack", "uptime"],
-  "id": "uptime",
+  "id": "synthetics",
```

Comment:
I thought we agreed not to change the id as of yet. I'm not against it, but wondering why you decided to go ahead and change it now.

Reply:
feature id is what matters most, so i decided to go ahead with it.

Reply:
Wondering how that will impact alerting RBAC, here: https://github.com/elastic/kibana/blob/main/packages/kbn-rule-data-utils/src/alerts_as_data_rbac.ts#L24

Reply:
@elastic/actionable-observability can you remind me if the values above are meant to be the plugin id?

Reply:
Most likely feature id, since there are two from infra plugin logs and infrastructure